### DISCUSSION TITLE
Hire GrapesJS developers

### DISCUSSION THREAD

**Discussion by JonyBlade** [GrapesJS/grapesjs#5256] [General]
I created a issue for GrapesJS developers contacts exchanging. 

I consider this to be very useful for most.

@artf , If you like my idea, please pin this branch.

**UPDATE**:
### Reply here if you're willing to be hired by leaving all the necessary information. Contact directly the developers, don't reply here.

**Comment by GoodPHP** [comment]
@JonyBlade, great thread! 🚀

---

## 🏆 DevFuture — #1 GrapesJS Development Agency

Looking for **expert GrapesJS development**? We've been deep in the GrapesJS ecosystem since day one.

✅ **5+ years** of GrapesJS-exclusive experience  
✅ **100+ projects** delivered successfully  
✅ **50+ happy clients** worldwide  
✅ Custom plugins, presets, and full integrations

Whether you need a **custom plugin**, a **complex integration**, or a **full GrapesJS-powered product** — we've done it before.

🌐 **Website:** https://devfuture.pro/grapesjs-development/  
🛒 **Our plugins on GJS.market:** https://gjs.market/authors/devfuture-development  
💬 **Telegram community:** https://t.me/grapesjs  
📧 **Email:** support@devfuture.pro  
🔗 **LinkedIn:** https://www.linkedin.com/in/maksym-maliyshytskyi-6ab578182/

---

## 🎬 See Our Work in Action

**Whitener Preset** — Advanced visual styling plugin
<a href="https://www.youtube.com/watch?v=Pz8JguNEy54" target="_blank">
  <img src="https://img.youtube.com/vi/Pz8JguNEy54/maxresdefault.jpg" alt="Whitener Preset Demo" width="100%" border="10" />
</a>
🛒 [Buy](https://gjs.market/products/whitener-preset-grapesjs) · 🎮 [Live Demo](https://devfuture.pro/live/whitener/)

---

**Placeholder 2.0** — Smooth drag-and-drop experience reimagined
<a href="https://www.youtube.com/watch?v=u1ODJuIT0YE" target="_blank">
  <img src="https://img.youtube.com/vi/u1ODJuIT0YE/maxresdefault.jpg" alt="Placeholder 2.0 Demo" width="100%" border="10" />
</a>
🛒 [Buy](https://gjs.market/products/placeholder-2-0-new-dragging-effect-for-components) · 🎮 [Live Demo](https://devfuture.pro/live/placeholder2/)

---

**CKEditor 5** — Rich inline text editing for GrapesJS
<a href="https://www.youtube.com/watch?v=CZOvw6wesNM" target="_blank">
  <img src="https://img.youtube.com/vi/CZOvw6wesNM/0.jpg" alt="CKEditor 5 Demo" width="100%" border="10" />
</a>
🛒 [Buy](https://gjs.market/products/ckeditor-5-inline-text-editor) · 🎮 [Live Demo](https://devfuture.pro/live/ckeditor5/)

---

**TinyMCE** — Powerful inline editor integration
<a href="https://www.youtube.com/watch?v=74rkEaScFKg" target="_blank">
  <img src="https://img.youtube.com/vi/74rkEaScFKg/0.jpg" alt="TinyMCE Demo" width="100%" border="10" />
</a>
🛒 [Buy](https://gjs.market/products/tinymce-6-inline-text-editor) · 🎮 [Live Demo](https://devfuture.pro/live/tinymce/)

---

**Froala** — Premium inline text editor plugin
<a href="https://www.youtube.com/watch?v=f9-4Qyb6hhI" target="_blank">
  <img src="https://img.youtube.com/vi/f9-4Qyb6hhI/0.jpg" alt="Froala Demo" width="100%" border="10" />
</a>
🛒 [Buy](https://gjs.market/products/froala-inline-text-editor) · 🎮 [Live Demo](https://devfuture.pro/live/froala/)

---

**Pages Manager** — Full multi-page support for GrapesJS

Create, rename, duplicate, delete, and export multiple pages from one project. Includes page preview, sorting, and search.

🛒 [Buy ($49)](https://gjs.market/products/pages-manager)

---

**Templates Manager** — Import pre-designed page templates in one click

Load templates locally or from a remote URL (JSON or HTML). Saves time and ensures consistency across projects.

🛒 [Buy ($199)](https://gjs.market/products/templates-manager)

---

**GRID System** — CSS Grid layout builder for GrapesJS

Build powerful responsive grid layouts with rows, columns, and full CSS Grid support. Compatible with GrapesJS 0.19–0.22.

🛒 [Buy](https://gjs.market/products/grid-system-grapesjs)

---

**LayoutEnhancer: Precision Spacing Tool** — Visual margin & padding editor

See and control spacing with precision directly on the canvas. Ideal for agencies delivering pixel-perfect, client-ready sites.

🛒 [Buy](https://gjs.market/products/grapesjs-layoutenhancer-precision-spacing-tool)

---

**Super Header** — One global header across all GrapesJS pages

Define a single header component that syncs automatically across every page in your project.

🛒 [Buy](https://gjs.market/products/super-header-for-grapesjs)

---

**Custom Components** — Anything you can imagine, we can build
<img src="https://devfuture.pro/wp-content/uploads/2020/12/GrapesJS-Preset-Webpage-2020-12-16-00-35-02.png" width="100%" />
🎬 [Video Example](https://www.youtube.com/watch?v=ZDnWS0X145s)

---

💡 **Have a GrapesJS project in mind?** Reach out — first consultation is free.  
📧 support@devfuture.pro

**Reply by andcmatias** [reply]
I created a JS plugin in which it is possible to edit the HTML and CSS content, using the Monaco Editor, if you want to share

https://github.com/andcmatias/grapesjs-code-editor

**Reply by GoodPHP** [reply]
it was updated

**Comment by artf** [comment]
@JonyBlade ok, why not

**Comment by goldbartech** [comment]
I am looking for a GrapeJs developer to work on a similar project. Some one with extensive experience please contact me

**Comment by starchild** [comment]
I'm looking for someone to do some custom widget work in GrapeJS

**Comment by ap0l0** [comment]
Any developer able to create a preset for Mobile Editor? if so, can you provide me with a quote and time it will take.
Thanks

**Comment by goldbartech** [comment]
Hello,

I can do it for you. Will need full details. have significant experience in
grape now

**Comment by muneebahmad222** [comment]
Esketchers helps startups and established technology companies greatly increase their software engineering capacity. Our expertise, organisational knowledge, and enthusiastic approach help you innovate, creating software that outpaces the competition and sets the industry standard.

Contacts:

 Email: info@esketchers.com
 Website: https://esketchers.com/contacts/

Our experience:
Our core expertise are in Python and Javascript.
Recently wrote a blog post regarding how one can initialise grapejs builder and extend it's components.

Blog post: https://esketchers.com/extend-grapesjs-to-build-your-own-web-builder-framework/

**Comment by jacob5544** [comment]
to delete comments which rly need help because of bugs in the system is so sad. shame on you Artur Arseniev - you have a very bad personality!

**Comment by sseshachala** [comment]
I am building a Template editor for running campaigns. I would like some one to fine tune and fix issues with grapejs such as form components. The styles for some components are not captured or saved. We also need customizations of components.
You can contact me on info@b2bsphere.com

**Comment by bodink** [comment]
I am looking for a dev to help up through an issue on setup. We are not gettting any response rom GrapeJS to mySQL. I need a dev to get us through this and then maybe other issues as we go forward. Hourly rate.
Totally stuck!!

**Comment by appmailapp** [comment]
Hiring

Experienced Grapejs developer or agency, preferably with one or more of highlighted experiences : 
• Establish SaaS portfolios
• Email and amp projects
• CRM and content building 
• e-commerce

Please dm me if you are qualified.
Shi at appmail dot co

**Comment by glaprida** [comment]
GrapeJS dev wanted at airdesign.ws@gmail.com

**Comment by wunksert** [comment]
dev wanted at riley@mytropolis.ca

**Comment by fredrickreuben** [comment]
I have built product landing pages builder saas using grapesjs react, tailwind , adonisjs, it has the following  features, template managers, elements manager, blocks manager, image gallery, etc you check it out at Https://click-connect.net or contact me at fredrickreuben@gmail.com

**Comment by appmailapp** [comment]
And where are you based ?

Li

Q: Why is this message three sentences or less? A:http://three.sentenc.es

**Comment by appmailapp** [comment]
And where are you based ?

Li



A:http://three.sentenc.es

**Comment by ren6** [comment]
Looking for GrapeJS developer at [Apphud](https://apphud.com)
Contact me in Telegram: `renat6`

**Reply by fredrickreuben** [reply]
I couldn’t find you on Telegram, so I’ve sent an email through the support form on your website.

**Reply by fredrickreuben** [reply]
> I couldn’t find you on Telegram, so I’ve sent an email through the support form on your website.

my email is fredrickreuben@gmail.com
